docs/source/developing_with_sdk/packaging_app.md

```diff
@@ -42,30 +42,34 @@ The file usually would look like below:
 The following list contains arguments that have default values or are provided through the SDK if these flags are not invoked.
 However, the user can choose to override these values by invoking these optional flags:
 
-* `--model, -m <MODEL_PATH>`: A path to local directory containing all application models (will override utilizing models provided within application SDK code).
-* `--log-level, -l <LEVEL>`: A logging level (`"DEBUG"`, `"INFO"`, `"WARN"`, `"ERROR"`, or `"CRITICAL"`).
-
-* `--base <BASE_IMAGE>`: A base docker image (overrides default `"nvcr.io/nvidia/pytorch:21.07-py3"`).
-* `--input-dir, -i <INPUT_DIR>`: A directory mounted in container for Application Input (overrides default `"input"`).
-* `--models-dir <MODELS_DIR>`: A directory mounted in container for Models Path (overrides default `"/opt/monai/models"`).
-* `--output-dir, -o <OUTPUT_DIR>`: A directory mounted in container for Application Output (overrides default `"output"`).
-* `--timeout <VALUE>`: A timeout (overrides default `0`. In seconds.).
-* `--version`: A version of the Application (overrides default `"0.0.0"`).
-* `--working-dir, -w <WORKDIR>`: A directory mounted in container for Application (overrides default `"/var/monai"`).
+Optional Path to the directory containing all application models
```

Comment:
Is this line supposed to be added here?

Reply:
Good catch! Thank you!

This one is a typo. Let me remove it.